edge_computing/ztp-sno-additional-worker-node.adoc

```diff
@@ -28,6 +28,8 @@ include::snippets/technology-preview.adoc[]
 
 * For more information about worker nodes, see xref:../nodes/nodes/nodes-sno-worker-nodes.adoc#nodes-sno-worker-nodes[Adding worker nodes to {sno} clusters].
 
+* For information about removing a {sno-caps} cluster, see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#auto-remove-host-steps-cli[Removing managed cluster nodes by using the command line interface].
```

Comment:
I wouldn't call this a process to "remove a cluster".

For a SNO without workers, unbinding would be sufficient. The process the linked annotation triggers is used to cleanly remove a worker node from a managed cluster.

So I'd say this should read "For information about removing a worker node from a {sno-caps} cluster..."